src/PathMatcher.php

```diff
@@ -0,0 +1,75 @@
+<?php declare (strict_types = 1);
+
+namespace OdbavTo\PresenterRoute;
+
+class PathMatcher
+{
+	/**
+	 * @var PathMatcher
+	 */
+	private static $instance;
+	
+	public static function getInstance(): PathMatcher
```

Comment:
It may seem nice and easy solution here, but it makes application very coupled in future years.
Instead of static singleton, I'd use service with interface as advised by @Lexinek bellow.